docker-compose.yml

```
@@ -51,5 +51,20 @@ services:
     environment:
       - WATCHPACK_POLLING=true
 
+  view-admin:
+    build:
+      context: .
+      dockerfile: .docker/view-admin.Dockerfile
+    container_name: "view-admin"
+    volumes:
+      - ./view-admin:/app
+    command: sh -c "npm install && npm run dev"
+    ports:
+      - "3001:3001"
```

```suggestion
      - "3001:3000"
```
ここcomitしてなかったね、これadd,comit,pushお願いします！